generator/src/test/java/com/scottlogic/deg/generator/cucumber/MatchingRegex.feature

```diff
@@ -407,107 +418,116 @@ Scenario: Running a 'matchingRegex' request alongside a non-contradicting shorte
        | null |
        | "b"  |
 
-@ignore
-Scenario: Running a 'matchingRegex' request alongside a contradicting shorterThan (equal) constraint should fail with an error message
-       Given foo is matching regex /[b]{2}/
+  @ignore #may need to raise issue
```

Comment:
Can we raise a bug on this or link to an existing?

Reply:
This is being addressed in ticket #501